Molio/Source/Domain/UseCase/FollowRelationUseCase/DefaultFollowRelationUseCase.swift

```diff
@@ -26,62 +30,75 @@ struct DefaultFollowRelationUseCase: FollowRelationUseCase {
         try await service.deleteFollowRelation(relationID: relationID)
     }
     
-    func fetchMyFollowingList() async throws -> [MolioFollowRelation] {
-        let userID = try await fetchMyUserID()
-        let relations = try await service.readFollowRelation(followingID: userID, followerID: nil, state: true)
-        return relations.map { relation in
-            MolioFollowRelation(
-                id: relation.id,
-                date: relation.date,
-                following: relation.following,
-                follower: relation.follower,
-                state: relation.state
-            )
-        }
+    func fetchMyFollowingList() async throws -> [MolioUser] {
+        let currentUserID = try await fetchMyUserID()
+        let followingRelations = try await service.readFollowRelation(followingID: currentUserID, followerID: nil, state: true)
+        
+        return try await fetchUsers(from: followingRelations.map { $0.follower })
     }
     
-    func fetchFreindFollowingList(userID: String) async throws -> [MolioFollowRelation] {
-        let relations = try await service.readFollowRelation(followingID: userID, followerID: nil, state: true)
-        return relations.map { relation in
-            MolioFollowRelation(
-                id: relation.id,
-                date: relation.date,
-                following: relation.following,
-                follower: relation.follower,
-                state: relation.state
-            )
-        }
+    func fetchFreindFollowingList(userID: String) async throws -> [MolioUser] {
```

Comment:
P5;
```suggestion
    func fetchFriendFollowingList(friendUserID: String) async throws -> [MolioUser] {
```

로 바꾸면 어떨까요?? 호출하는 코드에서 덜 헷갈릴 것 같습니다!
> `내 userID를 넣어야하나?` 라고 고민할 것 같습니다.

Reply:
맞아요! 그냥 일단 킵고잉 했지만, 수정해놓겠습니다

Reply:
함수명에 friend가 있어서 파라미터명은 그냥 `id`로도 충분해 보입니다!